kk/models/hearing.py

```diff
@@ -29,7 +30,7 @@ class Hearing(ModifiableModel):
     closed = models.BooleanField(verbose_name=_('Whether hearing is closed'), default=False)
     heading = models.TextField(verbose_name=_('Heading'), blank=True, default='')
     abstract = models.TextField(verbose_name=_('Abstract'), blank=True, default='')
-    heading = models.TextField(verbose_name=_('Content'), blank=True, default='')
+    content = models.TextField(verbose_name=_('Content'), blank=True, default='')
```

Comment:
This change was made by Raphaël earlier... It doesn't seem like you've rebased on top of the current master?

![image](https://cloud.githubusercontent.com/assets/58669/10941279/581782ae-8313-11e5-8aa0-c6f97c2f357f.png)